### PR TITLE
Disable_WiFi-MacOS

### PR DESCRIPTION
**Pull request by brunoooost**
Implementing the Disable WiFi - MacOS payload to the execution category on Hak5 - USB Rubber Ducky.

- **Disable WiFi - MacOS:** This payload is designed to turn off the Wi-Fi on a MacOS system.

### CODE REVIEW

File: payloads/library/execution/Disable WiFi - MacOS/payload.txt
Comment (hak5peaks) [inline payloads/library/execution/Disable WiFi - MacOS/payload.txt:1]: Please rename your directory and remove any spaces and replace them with `-` or `_` 


>Naming Conventions
Please give your payload a unique, descriptive and appropriate name. Do not use spaces in payload, directory or file names. Each payload should be submit into its own directory, with - or _ used in place of spaces, to one of the categories such as exfiltration, phishing, remote_access or recon. Do not create your own category.

**Review by hak5peaks — CHANGES_REQUESTED** [review]

**Comment by brunoooost** [comment]
done

**Comment by hak5peaks** [comment]
You changed the PR name. 

The payload directory name needs to be changed, currently its `Disable WiFi - MacOS` 

it needs to be `Disable_WiFi-MacOS` or `Disable-WiFi-MacOS` the directory name cannot contain spaces.

**Comment by brunoooost** [comment]
Now, yes

**Pull request merged**